tests/libcxx/tests.supported

```diff
@@ -4718,3 +4701,19 @@
 ../../3rdparty/libcxx/libcxx/test/support/test.support/test_poisoned_hash_helper.pass.cpp
 ../../3rdparty/libcxx/libcxx/test/support/test.workarounds/c1xx_broken_is_trivially_copyable.pass.cpp
 ../../3rdparty/libcxx/libcxx/test/support/test.workarounds/c1xx_broken_za_ctor_check.pass.cpp
+../../3rdparty/libcxx/libcxx/test/libcxx/containers/sequences/vector/vector.cons/construct_iter_iter.pass.cpp
```

Comment:
Are these the only new tests that have been added in 8.0.0? Just doing a quick scan, neither of these are accounted for in our test lists:
* test\libcxx\language.support\support.dynamic\libcpp_deallocate.sh.cpp
* test\std\containers\associative\multiset\insert_emplace_allocator_requirements.pass.cpp

I suspect there are probably others? Again, I would do a full recursive ls of the new test folder, sort the results and diff them against our tests.all to get a comprehensive view of everything that's changed.

Also, any new tests should be added to the tests.all list in addition to either tests.support/tests.unsupported/tests.broken.

Reply:
Rebuilt test lists with full recursive scan and reiterated until build was clean have a stable list of passing tests.